arch/arm/src/samv7/Kconfig

```
@@ -857,6 +857,14 @@ config SAMV7_PWM0_PA9_POL
 		Fault input is by default active on high level. This config
 		inverts the logic and makes fault active on low level.
 
+config SAMV7_PWM0_PA9_LATCHED
+	bool "Latched fault"
+	default n
+	---help---
+		This options ensures the fault input remains active until it is cleared
+		by the software. Otherwise, the fault is cleared once the fault input
+		changes its polarity level back to normal state.
```

At minimum (extraneous `s` in `options`):

```suggestion
		This option ensures the fault input remains active until it is cleared
```

Perhaps for clarity:

```suggestion
		When this option is selected, the fault input remains active until it
		is cleared by the software. When this option is not selected, the
		fault is cleared once the fault input changes its polarity level back
		to normal state.
```

WDYT?

---

Yes, that looks more comprehensible, changed.